python/cudf/cudf/core/udf/_ops.py

```diff
@@ -0,0 +1,20 @@
+import operator
+
+arith_ops = [
```

Comment:
I think we should add everything we can that will just work if we add it to this list as well as the tests, which ones am I missing?

Reply:
From looking at Numba's implementation of the `operator` module, I see the following that would need to be tested / checked:

Bitwise operations (for integer types):

```python
bit_ops = ( 
    operator.lshift,
    operator.rshift,
    operator.and_,
    operator.or_,
    operator.xor,
)
```

Inplace operations:

```
inplace_ops = ( 
    operator.iadd,
    operator.isub,
    operator.imul,
    operator.imod,
    operator.itruediv,
    operator.ifloordiv,
    operator.ipow,
    operator.ilshift,
    operator.irshift,
    operator.iand,
)
```

Most of the unary operations:

```
unary_ops = ( 
    operator.truth,
    operator.invert,
    operator.neg,
    operator.pos,
    operator.not_
)
```

Maybe some instances of `is` and `is not` from the comparison operations:

```
comparison_ops = (
    operator.lt,
    operator.le,
    operator.eq,
    operator.ne,
    operator.ge,
    operator.gt,
    operator.is_,
    operator.is_not,
)
```

Some of the unary ops:

```
unary_ops = (
    operator.truth,
    operator.invert,
    operator.neg,
    operator.pos,
    operator.not_
)
```

I think it's fairly straightforward to add everything, but needs some thought / care to make sure that we only provide typing for things that make sense - e.g. not providing bitwise operations on floats. I do still think it would be better to get this over the line then add more in another PR than to add to the workload of this PR (and I'm happy to help with the addition of more when I get a chance).

Reply:
Great summary. I'll make sure to file some issues for these once this merges. 

When we don't support something, should we just not provide typing for those cases, and let the typing machinery raise the standard numba error? Or would it be better to provide typing that detects the scenario and then errors with something more specific (I assume this would bubble up through numba's normal traceback system anyways) - just thinking forward for disabling cases like `__pow__` and some of the other functions we have problems with currently